### ISSUE TITLE
A line in main.py().

### ISSUE THREAD

**Issue by debasmitdas** [SSARCandy/DeepCORAL#10]
What is the use of model.train() in line 25 of main.py ? Is it not redundant ?

**Comment by SSARCandy** [comment]
Yes but explicitly state in train mode would be more readable.
